src/panel/direction/DirectionForm.jsx

```diff
@@ -11,15 +11,14 @@ export default class DirectionForm extends React.Component {
     destination: PropTypes.object,
     onChangeDirectionPoint: PropTypes.func.isRequired,
     onReversePoints: PropTypes.func.isRequired,
+    onEmptyOrigin: PropTypes.func.isRequired,
```

Comment:
These two functions are required… but unused. Looks like obsolete WIP code you forgot to remove. (And it reminds me we can add an ESLint rule to highlight that)